pkg/runtime/kubernetes.go

```diff
@@ -3,3 +3,7 @@ package runtime
 func (k kubernetes) getRuntime() string {
 	return "kubernetes"
 }
+
+func (k kubernetes) shouldJoinNetworks() (bool, PostSandboxCreate, PostSandboxDestroy) {
+	return false, nil, nil
```

Comment:
Confirming for my own edification -- this is strictly an Openshift feature?

Reply:
The openshift plugin for the multitenant environment is only for openshift. 

The network policy stuff is for both k8s and openshift so that is apart of the actual create sandbox and destroy sandbox methods.

Reply:
Thanks.